components/list/list-item-generic-layout.js

```diff
@@ -390,6 +414,17 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		this._cellNum = parseInt(slot.getAttribute('data-cell-num'));
 		this._cellFocusedItem = this._getFocusedItemPosition(event.target);
 	}
+
+	_unclickActionSlot(event) {
+		const related = event.relatedTarget;
+		const slot = (event.path || event.composedPath()).find(node =>
+			node.nodeName === 'SLOT' && node.classList.contains('d2l-cell'));
+		if (!related || !isComposedAncestor(slot, related)) {
+			setTimeout(() => {
+				this._actionFocused = false;
+			}, 200);
```

Comment:
Timeout ensures animations have time to process

Reply:
Can you use the animation end event to get a better timer?

Reply:
Which animation are you waiting for?  I agree with Maya here - it would be best to wait for an animation or transition event, but be mindful of components that support prefers reduced-motion, in such case there will be no-animation and therefore no event.  We have a few examples in core that check this in JS.

Reply:
Yup, that is a better solution, I agree.

Reply:
This is now a moot point because they are hooked into events instead.